code/src/mlx90640_processor.cpp

```diff
@@ -0,0 +1,105 @@
+#include <mlx90640_processor.hpp>
+
+namespace r2d2::thermal_camera {
+
+    mlx90640_processor_c::mlx90640_processor_c(mlx90640_i2c_c &bus)
+        : bus(bus), Kvdd(0), Vdd25(0) {
+    }
+
+    void mlx90640_processor_c::set_and_read_raw_pixels() {
```

Comment:
Any reason for this to be empty?

Reply:
Function is not implemented yet. It's still on the TO-DO.